client/css/style.css

```diff
@@ -681,19 +665,58 @@ button {
 #chat.hide-quit .quit {
 	display: none !important;
 }
-#chat .notice .type {
-	display: none;
+#chat .join .from:before {
+    font-family: FontAwesome;
+	content: "\f090";
+	color: #2ECC40;
+}
+#chat .kick .from:before {
+    font-family: FontAwesome;
+	content: "\f00d";
+	color: #FF4136;
+}
+#chat .part .from:before,
+#chat .quit .from:before {
+    font-family: FontAwesome;
+	content: "\f08b";
+	color: #FF4136;
+}
+#chat .topic .text,
+#chat .topic .user {
+	color: #2ECC40 !important;
+}
+#chat .topic .from:before {
+    font-family: FontAwesome;
+	content: "\f0a1";
+	color: #2ECC40;
+}
+#chat .mode .from:before {
+    font-family: FontAwesome;
+	content: "\f013";
```

Comment:
This one is tricky... But we can't use the `cog` here as it's usually used for settings, and we actually use it for that reason in the app.
Possible alternatives:
- https://fortawesome.github.io/Font-Awesome/icon/info-circle/
- https://fortawesome.github.io/Font-Awesome/icon/key/
- https://fortawesome.github.io/Font-Awesome/icon/terminal/
- https://fortawesome.github.io/Font-Awesome/icon/info/